### ISSUE TITLE
Bot doesn't say anything when it dispo's certain calls

### ISSUE THREAD

**Issue by NicksAccount** [MTS11648/Don-NonAutomated-Bot#32]
#### Brief description of the issue:
On some of the shortcuts for some questions, (for example, "2 - Correct customer unavailable") there is no "Ok, have a good day" or "Alright, thank you" before the call gets hungup.
#### What happened?:
Call hungup and dispo'd without saying goodbye or something along those lines
#### What _should_ have happened?:
"Alright, thank you." hangup
#### Steps to reproduce:
Press 2 on first question on manual bot
#### Additional info:
Not sure if it's cause there wasn't a file or not. But, there is now

**Comment by SpadesNeil** [comment]
I think it's because we lacked a file. @MTS11648 will know for sure.

**Comment by MTS11648** [comment]
no, we're not lacking an audio clip, the thing is the audio clip begins to play then it immediately hangs up and stops the audio clip.
